.travis.yml

```diff
@@ -21,6 +21,7 @@ cache:
 
 script:
  - npm run test
+ - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then echo "Your live demo url, if everything went well, should be https://coveo.github.io/react-vapor/$(TRAVIS_PULL_REQUEST_BRANCH)/"; fi' # run this line on pull request builds only
```

Comment:
Any reason we aren't using `== "true"` instead of `!= "false"`?

Reply:
IIRC I copied this from the travis docs so i just did not bother trying `== "true"`